ShortestPath/ShortestPath/NetworkBuilder.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace ShortestPath
+{
+    public class NetworkBuilder
+    {
+        public void BuildNetwork(Network network, List<Arc> arcs)
```

Comment:
NettworkBuilder.BuildNetwork should return an object of type Network, not mofifying an input parameter.

Reply:
I think Chosen names for this class and its method were wrong, because Network object was created in Program class and in this class it's going to be assigned not be created .
NetworkBuilder -> NetworkAssigner (class)
NetworkBuild -> NetworkAssign (metthod)